src/doc/guide.md

```diff
@@ -169,17 +160,17 @@ fn main() {
 }
 ```
 
-These lines define a **function** in Rust. The `main` function is special:
-it's the beginning of every Rust program. The first line says "I'm declaring a
+These lines define a **function** in Rust. The `main` function is special: it's
+the beginning of every Rust program. The first line says "I'm declaring a
 function named `main`, which takes no arguments and returns nothing." If there
-were arguments, they would go inside the parentheses (`(` and `)`), and because
-we aren't returning anything from this function, we've dropped that notation
-entirely.  We'll get to it later.
+were arguments, they would go inside the parentheses, and because we aren't
```

Comment:
It's more consistent with the rest of the docs, which do mention words for symbols upon seeing them the first time.